README.md

```diff
@@ -22,6 +22,27 @@ $ ./docker_install_ubuntu.sh
 
 # Quickstart using Docker-Compose
 
+## Prerequisites
+
+This repo has a directory called 'volttron', which contains the volttron codebase. In other words, this repo contains another repo in a subfolder. 
```

Comment:
proper terms...git submodule (another repository)

Please use repository not repo